bin/make/functional_tests_orm.mk

```diff
@@ -8,8 +8,8 @@ functional_tests_orm:
 	@${CONSOLE} doctrine:schema:create --env=orm
 	@echo '+++ run ORM functional tests +++'
 ifeq ($(HAS_XDEBUG), 0)
-	@vendor/bin/simple-phpunit --prepend build/xdebug-filter.php -c phpunit.xml.dist --coverage-clover build/logs/clover.xml --testsuite "functional tests with orm"
```

Comment:
the php file was not existing, and `--prepend` is only available from phpunit 7 on.

explicitly specifying the .dist phpunit.xml file prevents people from running locally with a customized phpunit.xml file.